lib/task/import/csvImportTask.class.php

```diff
@@ -1056,6 +1056,13 @@ public function execute($arguments = array(), $options = array())
         {
           $this->handleDigitalObjectPath($self, $path);
         }
+
+        // If importing a translation, re-save entire information object so translation will be indexed
+        if ($self->object instanceof QubitInformationObjectI18n && !$search->searchIndexingDisabled)
```

Comment:
Does this respect the --index flag? I can't see where $search is getting set.  

The index flag is assigned to the $import object here:
https://github.com/artefactual/atom/blob/qa/2.6.x/lib/task/import/csvImportTask.class.php#L1062

Reply:
Oops... nice catch! Fixed!